indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -64,17 +64,17 @@
     background-color: lighten($dark-gray, $color-variation);
     cursor: move;
 
-    &:before,
-    & > .handle:before {
+    &::before,
+    & > .handle::before {
```

Comment:
you don't need the `&` here. You can use `> .handle::before`. Applies in other places below.
I've added it to the [SASS Guidelines](https://github.com/indico/indico/wiki/SASS-Guidelines). /cc @nop33